guardian/about-us
# Dependency upgrades

## What does this change?

### Dependency upgrades

- "@emotion/react": "^11.10.6"
- "@guardian/source-foundations": "^10.0.1"
- "@guardian/source-react-components": "^13.0.0"
- "next": "^13.3.0"
- "react": "^18.2.0",
- "react-dom": "^18.2.0"
- "@types/react": "^18.2.0"
- "@guardian/libs": "^14.0.0"
- "@guardian/eslint-plugin-source-foundations": "^11.0.0",
- "@guardian/eslint-plugin-source-react-components": "^15.0.0",
- "@guardian/consent-management-platform": "^13.0.0",
- removes "next-transpile-modules" as it's now [integrated with Next 13](https://github.com/martpie/next-transpile-modules/releases/tag/the-end)

### Other changes

- Tidies up some unused imports and `jsx` pragmas
- Tidies the README
- Adds a README section on a known issue affecting the dev server only
  - when running the dev server, there is an error in the browser's console "Prop `className` did not match" error in the browser console
  - this does not affect dev server functionality, and the error does not appear in CODE. Given this context, we (@OllysCoding) decided to make a note

## What comes next

- Add a linter
- We could move this to [`guardian/csnx`](https://github.com/guardian/csnx)!

## How to test

- Because this project doesn't have automated testing, we should manually verify the change between local (or [CODE](https://gu-about-us.s3-eu-west-1.amazonaws.com/CODE/gu-about-us-upload/about/index.html)) and PROD before merging
  - annoyingly, images do not work on CODE both with the current `main` or this branch. They do work locally though. I think we could assume that they will also work on PROD.
